Cargo.toml

```diff
@@ -32,8 +33,12 @@ cw-controllers = { version = "1.0.1" }
 cw-utils = { version = "1.0" }
 cw-storage-plus = "1.0.1"
 cw2 = { version = "1.0.1" }
-cw20 = { version = "1.0.1" }
-cw20-base = { version = "1.0.1", features = ["library"] }
+cw20-base = { git = "https://github.com/AbstractSDK/cw-plus.git", features = [
+  "interface",
+], branch = "update/change_macro_name" }
+cw20 = { git = "https://github.com/AbstractSDK/cw-plus.git", features = [
+  "interface",
+], branch = "update/change_macro_name" }
```

Comment:
When using branch dependencies like this it's nice to also mention that in the PR description. Most of the time you want both branches to be merged at the same time so that the branch specifier can be removed.